pytest_pyodide/hook.py

```diff
@@ -31,6 +31,26 @@
 RUNTIMES_NO_HOST = [f"{runtime}-no-host" for runtime in RUNTIMES]
 
 
+class PytestWrapper:
```

Comment:
We will probably need to find a way to share the state in a better way. But for now it should be fine.